mantidimaging/gui/windows/main/view.py

```diff
@@ -34,14 +36,19 @@ class MainWindowView(BaseMainWindowView):
     NOT_THE_LATEST_VERSION = "This is not the latest version"
     UNCAUGHT_EXCEPTION = "Uncaught exception"
 
-    active_stacks_changed = Qt.pyqtSignal()
-    backend_message = Qt.pyqtSignal(bytes)
+    active_stacks_changed = pyqtSignal()
+    backend_message = pyqtSignal(bytes)
+
+    menuFile: QMenu
+    menuWorkflow: QMenu
+    menuImage: QMenu
+    menuHelp: QMenu
 
     actionRecon: QAction
     actionFilters: QAction
     actionSavuFilters: QAction
     actionCompareImages: QAction
-    actionLoadLog: QAction
+    actionSampleLoadLog: QAction
```

Comment:
This can load logs for other things.. we don't use it but there's also logs for the flat images for #500

Reply:
I realize that, but that is the actual name of the action, and actionLoadLog points to nothing in the UI file.

Reply:
Ah I see